test/data/matpower/extra_components.m

```diff
@@ -0,0 +1,67 @@
+%% MATPOWER Case Format : Version 2
+mpc.version = '2';
```

Comment:
case3.m is nominally the place to test the extended Matpower files.  Can you extend case3.m to cover your cases instead of adding a new case?